examples/kyb-app/src/domains/collection-flow/collection-flow.api.ts

```diff
@@ -61,7 +65,18 @@ export const fetchActiveWorkflow = async (dto: GetActiveWorkflowDto): Promise<Fl
     })
     .json<{ result: FlowData }>();
 
-  return result.result;
+  const flowData = result.result;
+  const filesMetadata = extractFilesMetadata(flowData, dto.documentConfigurations);
+
+  const files = await downloadAndBuildFiles(filesMetadata);
```

Comment:
It seems `kyb-app` has many places which were supposed to use `react-query`. Not sure if an in memory store is needed between the URL, global React state, and `react-query`.